openpype/hosts/nuke/plugins/publish/validate_write_nodes.py

```diff
@@ -105,28 +112,33 @@ def process(self, instance):
                 )
 
             key = knob_data["name"]
-            value = knob_data["value"]
+            values = values_by_name[key]
```

Comment:
I'd probably change these to more meaningful names.

Reply:
Got any suggestions?